integration/test/metric_value_benchmark/mem_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build linux && integration
+// +build linux,integration
+
+package metric_value_benchmark
+
+import (
+	"github.com/aws/amazon-cloudwatch-agent/integration/test/metric"
+	"github.com/aws/amazon-cloudwatch-agent/integration/test/status"
+	"time"
+)
+
+type MemTestRunner struct {
+}
+
+var _ ITestRunner = (*MemTestRunner)(nil)
+
+func (m *MemTestRunner) validate() status.TestGroupResult {
+	metricsToFetch := m.getMeasuredMetrics()
+	testResults := make([]status.TestResult, len(metricsToFetch))
+	for i, name := range metricsToFetch {
+		testResults[i] = m.validateMemMetric(name)
+	}
+
+	return status.TestGroupResult{
+		Name:        m.getTestName(),
+		TestResults: testResults,
+	}
+}
+
+func (m *MemTestRunner) getTestName() string {
+	return "Mem"
+}
+
+func (m *MemTestRunner) getAgentConfigFileName() string {
+	return "mem_config.json"
+}
+
+func (m *MemTestRunner) getAgentRunDuration() time.Duration {
+	return minimumAgentRuntime
+}
+
+func (m *MemTestRunner) getMeasuredMetrics() []string {
+	return []string{
+		"mem_active", "mem_available", "mem_available_percent", "mem_buffered", "mem_cached",
```

Comment:
nit: I don't think this gets called more than once but if it does we can pull the constant out?

Reply:
We can. I kind of hate the idea of pulling this out as a constant for every plugin test. We'd have to do something like:
```
var memMetrics = []string{}
var cpuMetrics = []string{}
var diskioMetrics = []string{}
```
and have to deal with all of these constants being prefixed by their plugin name, when they're only ever referenced in an implemented function.

Reply:
yeah and also this only ever gets called once.  And yeah that is redundant. Forgot we had that convo.